janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -168,6 +168,11 @@
     public static final ConfigNamespace ES_CREATE_EXTRAS_NS =
             new ConfigNamespace(ES_CREATE_NS, "ext", "Overrides for arbitrary settings applied at index creation", true);
 
+    public static final ConfigOption<Boolean> USE_EXTERNAL_MAPPINGS =
+            new ConfigOption<Boolean>(ES_CREATE_NS, "use-external-mappings",
+            "Whether JanusGraph should make use of an external mapping when registering an index." +
+            "Enabling this option make JanusGraph use an external mapping.", ConfigOption.Type.MASKABLE, false);
```

Comment:
Remove the second sentence ("Enabling this option...") or else add a space after the preceding period and change "make" to "makes".